utils/run-test

```diff
@@ -171,6 +171,7 @@ def main():
         # build_dir.
         paths = args.paths
         if not paths:
+            parser.print_usage()
             error_exit("error: too few arguments")
```

Comment:
Agreed, but I didn't want to change the output status of the command in case it's being used by something else.